### ISSUE TITLE
different sized popups

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#290]
I just been playing with the latest version of popups and have noticed a curious effect (this may have been in previous incantations)
If I request the page `en.wikiquote.org/wiki/Patience` and scroll down a bit 
![popup0](https://user-images.githubusercontent.com/861631/49071302-8f7f5a80-f225-11e8-9f3c-1c3026b0af64.jpg)
and hover over 'Patience', I get
![popup1](https://user-images.githubusercontent.com/861631/49071341-a756de80-f225-11e8-840b-4ce1e8a2346a.jpg)
If I move the mouse away (and the box disappears), then hover over again
![popup2](https://user-images.githubusercontent.com/861631/49071389-c2c1e980-f225-11e8-9cb0-e7c45ce046a3.jpg)

Curious?

Going off topic slightly, would it be possible to arrange that the popup does not obscure the link. For example in the above page, there is plenty of real estate to the left

**Comment by gnosygnu** [comment]
Yeah, I changed the positioning logic a bit in the latest version. 

It looks like the above can be handled by two changes:
* Add some sort of min-width (to avoid the narrow box)
* Increase the magic-number for offset-y

Much of this logic should now be in either of these two functions:
* CreatePopup: https://github.com/gnosygnu/xowa/blob/master/res/bin/any/xowa/html/res/src/xowa/popups/xo.popup.js#L234
* UpdatePopupPos:, https://github.com/gnosygnu/xowa/blob/master/res/bin/any/xowa/html/res/src/xowa/popups/xo.popup.js#L322

I'll take a look at this sometime soon as it should be simple changes.

Thanks.
